src/main/java/com/denizenscript/clientizen/tags/EntityTag.java

```diff
@@ -0,0 +1,80 @@
+package com.denizenscript.clientizen.tags;
+
+import com.denizenscript.denizencore.objects.Fetchable;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.tags.ObjectTagProcessor;
+import com.denizenscript.denizencore.tags.TagContext;
+import com.denizenscript.denizencore.utilities.debugging.Debug;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.entity.Entity;
+
+import java.util.UUID;
+
+public class EntityTag implements ObjectTag {
+
+	public UUID uuid;
+
+	public EntityTag(UUID uuid) {
+		this.uuid = uuid;
+	}
+
+	public EntityTag(Entity entity) {
+		this(entity.getUuid());
+	}
+
+	@Fetchable("e")
+	public static EntityTag valueOf(String string, TagContext tagContext) {
+		if (string == null) {
+			return null;
+		}
+		if (string.startsWith("e@")) {
+			return new EntityTag(UUID.fromString(string.substring(2)));
```

Comment:
Should check if it starts with `e@` and substring before doing the UUID conversion, as just a UUID is valid as well